Tests/KeyboardManagerTests.swift

```diff
@@ -109,40 +111,38 @@ fileprivate extension KeyboardManagerTests {
 
     func postTestNotification(name: Notification.Name) {
         notificationCenter.post(name: name, object: nil, userInfo: [
-                UIKeyboardFrameEndUserInfoKey: NSValue(cgRect: endFrame),
-                UIKeyboardFrameBeginUserInfoKey: NSValue(cgRect: beginFrame),
-                UIKeyboardAnimationDurationUserInfoKey: NSNumber(floatLiteral: animationDuration),
-                UIKeyboardIsLocalUserInfoKey: NSNumber(booleanLiteral: isLocal),
-                UIKeyboardAnimationCurveUserInfoKey: curve
+            UIKeyboardFrameEndUserInfoKey: NSValue(cgRect: endFrame),
+            UIKeyboardFrameBeginUserInfoKey: NSValue(cgRect: beginFrame),
+            UIKeyboardAnimationDurationUserInfoKey: NSNumber(floatLiteral: animationDuration),
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/alphatroya/KeyboardManager/blob/6d60cbed1341e5ddda2d31aeda6b5a2def6c00e2/Tests/KeyboardManagerTests.swift#L116"></span>The initializers declared in compiler protocols such as <code>ExpressibleByArrayLiteral</code> shouldn’t be called directly.</td>
    </tr>
  </tbody>
</table>